dev/com.ibm.ws.security.auth.data.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -37,9 +37,3 @@ maxSize.desc=Maximum number of entries supported by the authentication data subj
 # Do not translate: Kerberos, principal
 krb5Principal=Kerberos service principal name
 krb5Principal.desc=The name of the principal to be used. The principal can be a simple username or a service name.
```

Comment:
krb5Principal.desc=The name of the Kerberos principal name or Kerberos service name to be used.